src/components/editTranscript/EditTranscript.tsx

```diff
@@ -13,13 +13,24 @@ import {
   Text,
 } from "@chakra-ui/react";
 
-import MdEditor, { ExposeParam } from "md-editor-rt";
 import "md-editor-rt/lib/style.css";
+import MdEditor from "react-markdown-editor-lite";
+// import sanitize from "sanitize-html";
```

Comment:
we should remove this since we are not using the library anymore

Reply:
also remove it as a dependency

Reply:
The new commit fixes this